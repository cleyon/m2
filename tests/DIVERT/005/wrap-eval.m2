Hello
@define X 1
@wrap Wrapped text here...  X should be 2, but actually @X@
@incr X
X=@X@
Wrapped text follows:
