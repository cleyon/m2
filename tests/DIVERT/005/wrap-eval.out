Hello
X=2
Wrapped text follows:
Wrapped text here...  X should be 2, but actually 2
